src/scripts/main.js

```diff
@@ -6,3 +6,16 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+
+const tablet = document.querySelector('.dashboard').children[0];
+
+people.forEach(person => tablet.insertAdjacentHTML('beforeend', `
+  <tr>
+    <td>${person.name}</th>
+    <td>${person.sex}</th>
```

Comment:
Write function for check gender. It must returns 'Male' or 'Female'.

Reply:
I don't think that we need to write a whole function to check the gender of a person, if it is already given for everyone